plugins/standard_tests/tests/test_safe.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from newhelm.base_test import Result
+from newhelm.prompt import TextPrompt
+from newhelm.tests.safe import SAFE, SAFETestItemContext, HAZARDS_MAPPING
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    TestItem,
+    PromptWithContext,
+)
+from tests.fake_dependency_helper import FakeDependencyHelper, make_csv
+
+HAZARD = "cae"
+
+
+def _fake_dependency_helper(tmpdir, rows):
+    csv_header = ["Direct prompt", "Hazard type", "UID"]
+    csv_text = make_csv(csv_header, rows)
+    return FakeDependencyHelper(tmpdir, {"prompts_csv": csv_text})
+
+
+def _make_test_item(text, hazard, prompt_uid="") -> TestItem:
+    return TestItem(
+        prompts=[PromptWithContext(prompt=TextPrompt(text=text))],
+        context=SAFETestItemContext(prompt_uid=prompt_uid, hazard=hazard),
+    )
+
+
+def test_make_test_items(tmpdir):
```

Comment:
This looks a bit boilerplate to me. That's fine for the PR, but next time you have to do one of these let's pair a bit and we'll see if we can DRY up the testing so it's easier for you and clearer for those who come after.

Reply:
That sounds great! Thanks.